examples/nlp/text_classification/config.yaml

```diff
@@ -0,0 +1,58 @@
+# Sentence classification with pretrained BERT models
+batch_size: 32
+
+# IO
+work_dir: ./outputs
+checkpoint_dir: null
+
+# Data
+data_dir: /raid/data/nlp/SST-2 #/path/to/data
+train_file_prefix: train
+eval_file_prefix: dev
+shuffle: true # shuffle training data
+num_train_samples: -1
+num_val_samples: -1
+num_workers: 2
+pin_memory: false
+use_cache: true
+
+# Language Model
+pretrained_model_name: roberta-base
+bert_checkpoint: null
+bert_config: null
+# TODO: we can specialize config based on other config values at runtime:
+# https://hydra.cc/docs/next/patterns/specializing_config
+tokenizer: nemobert # only used if using custom checkpoint
+tokenizer_model: null # only used if tokenizer is sentencepiece
+do_lower_case: null # true for uncased models, false for cased models
+max_seq_length: 36
+
+# Classifier
+num_output_layers: 2
+fc_dropout: 0.1
+class_balancing: null # or weighted_loss
+
+# Pytorch Lightning
+pl:
```

Comment:
Why do we need this extra level of pl?

Reply:
PyTorch lightning may have other things we want to configure. For now it's just trainer.

Reply:
We can revise it later but I think we should avoid adding extra level as long as it is not super necessary. These levels can be confusing when working with command arguments.